dev-libs/libxlsxwriter/files/libxlsxwriter-1.1.4-fix-pkgconfig-version.patch

```diff
@@ -0,0 +1,17 @@
+Patch greedy dot star to fix mismatched pkg-config file version.
```

Comment:
If it's fixed upstream, can you give a link to the fix there?

Reply:
https://github.com/jmcnamara/libxlsxwriter/commit/12afb351db634ed9452eb2f38d68883a9538b8ef Here it is

Reply:
Now pop it at the top of the patch :)

Reply:
We always want to put all possible references at the top of patches: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#how.

Reply:
Whew that took long. Git rebase is hard sometimes. I added it to the top of the patch, I hope it's not terrible formatting.

Reply:
Nice!